go.mod

```diff
@@ -1,14 +1,3 @@
 module github.com/aquasecurity/libbpfgo
 
-go 1.18
-
-require (
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/sys v0.0.0-20210514084401-e8d321eab015
-)
-
-require (
-	github.com/davecgh/go-spew v1.1.0 // indirect
-	github.com/pmezard/go-difflib v1.0.0 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
-)
+go 1.19
```

Comment:
@grantseltzer This went unnoticed. I didn't see any Go version bump information in the main PR comment.